components/chef-automate-collect/chef-automate-collect.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
The Golang convention is to use `_` in the filename instead of `-`. Will you update this and the other files that have dashes in them?

Reply:
Done